api/src/routing.js

```diff
@@ -189,7 +189,7 @@ app.get('/favicon.ico', (req, res) => {
   });
 });
 
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, '../build/public')));
 app.use(express.static(path.join(__dirname, 'extracted-resources')));
```

Comment:
Are extracted resources being moved to `build`?

Reply:
Yes that's my plan but there are other PRs that touch that so to avoid conflicting I'm going to hold off and make that change later.